env/initialize.sh

```diff
@@ -213,12 +213,23 @@ initialize_chromedriver() {
   fi
   log_message "Installing chromedriver..."
   rm -rf ./chromedriver
-  # Determine the currently installed version of Chrome. Leave only the numbers,
-  # e.g., 72.0.3626.0.
-  local chrome_version=$(google-chrome --version | cut -f 3 -d ' ')
-  # Download Chromedriver. As of now, the maintainers guarantee that there's a
-  # matching Chromedriver artifact for every publicly released Chrome version.
-  local chromedriver_url="https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/${chrome_version}/linux64/chromedriver-linux64.zip"
+  # Determine the currently installed version of Chrome, e.g., "72".
+  # The program's output format is "Google Chrome 72.0.3626.0", of which we take
+  # the third space-separated token and grab its first dot-separated part.
+  local chrome_milestone=$(google-chrome --version | cut -f 3 -d ' ' |
+    cut -f 1 -d '.')
+  # Download Chromedriver's releases information.
+  local chromedriver_releases_url="https://googlechromelabs.github.io/chrome-for-testing/latest-versions-per-milestone-with-downloads.json"
+  if ! wget "${chromedriver_releases_url}" ; then
+    log_error_message "Failed to fetch chromedriver releases at ${chromedriver_releases_url} ; skipping chromedriver installation..."
+    return
+  fi
+  # Parse the JSON to get the needed milestone's Chromedriver download URL.
+  local chromedriver_url=$(jq -r ".milestones[] | \
+    select(.milestone == \"${chrome_milestone}\") | .downloads | \
```

Comment:
you can also use `https://googlechromelabs.github.io/chrome-for-testing/LATEST_RELEASE_{milestone}` or with first three numbers like `72.0.3626` instead of milestone.
is `jq` supposed to be installed by default?

Reply:
Good point, let me try it out. I think originally they wanted to kill these convenience URLs together with the switch to the new servers, but seems like they brought it back 2 weeks ago: https://github.com/GoogleChromeLabs/chrome-for-testing/pull/44

Reply:
So, lol, write-scratch-write-again strategy seems to be actual again! (context: #834 which stopped using `LATEST_RELEASE_`)

Reply:
🤯

Reply:
Done `https://googlechromelabs.github.io/chrome-for-testing/LATEST_RELEASE_`. So essentially it's a revert of #834, with the exception of different domains and modulo comments.